package.json

```diff
@@ -20,10 +20,20 @@
   },
   "devDependencies": {
     "@types/tape": "^4.2.34",
-    "tap-min": "^2.0.0",
-    "tape": "^4.13.0",
+    "ava": "^3.8.2",
     "ts-node": "^8.6.2",
     "tslint": "^5.8.0",
     "typescript": "^3.5.3"
+  },
+  "ava": {
+    "extensions": [
+      "ts"
+    ],
+    "files": [
+      "tests/**/*.test.ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ]
```

Comment:
[Q] Does AVA support somthing like `ava.config.json`? I'd rather not pollute the package.json. Although that would me more files in the root but 🤷‍♂️

Reply:
It does, can switch it to that!